### ISSUE TITLE
Could not load type WebDriverWait

### ISSUE THREAD

**Issue by SvenAelterman** [TestStack/TestStack.Seleno#262]
When using Selenium 3.5.2 or higher, I get this exception:
Could not load type 'OpenQA.Selenium.Support.UI.WebDriverWait' from assembly 'WebDriver.Support, Version=3.6.0.0, Culture=neutral, PublicKeyToken=null'

Based on the stacktrace, the exception is triggered by ElementFinder.cs line 24.

I can't find anything in the Selenium release notes that would indicate a potential cause.

**Comment by jimmymain** [comment]
that class is no longer in WebDriver.Support

**Comment by SvenAelterman** [comment]
Well, that would explain the issue. 

I have the feeling Seleno's been largely abandoned as a project?

**Comment by jimmymain** [comment]
Let’s hope not
I am contributing to some other projects at the mo
Can’t help right now - but it’s a great abstraction - let’s hope it continues...

**Comment by SvenAelterman** [comment]
@jimmymain I agree. I am trying to resolve, but the dependencies are all so out of date, it's hard to know where to begin :(

**Comment by jimmymain** [comment]
I installed pre 5.2.1 and it works fine

**Comment by robdmoore** [comment]
I wouldn’t say abandoned, but certainly all of the core maintainers have other focus areas and haven’t been investing time into it.

People still seem to be getting use out of it. If members of the community want to step up to inject some love we’d be happy to grant access to edit the repo.

**Comment by SvenAelterman** [comment]
@jimmymain I don't understand "pre 5.2.1"? Version 5.2.1 of what?

**Comment by SvenAelterman** [comment]
So about two years later, and I've finally updated TestStack.Seleno. If anyone would like to try it out, it's [here](https://github.com/Sorrell-Solutions/TestStack.Seleno).

I haven't update the .nuspec yet, and there are still some NuGet packages out of date. However, Seleno now works with the latest Selenium (3.141).

@robdmoore I will submit a PR, but be aware it's a huge changeset.

**Comment by SvenAelterman** [comment]
@robdmoore Update: PR submitted: https://github.com/TestStack/TestStack.Seleno/pull/267
